commands_text/solve_problem.go

```diff
@@ -0,0 +1,74 @@
+package commands_text
+
+import (
+	"github.com/assistant-ai/jess/prompt"
+	"github.com/urfave/cli/v2"
+)
+
+type SolveProblem struct{}
+
+func (c *SolveProblem) Name() string {
+	return "solve"
+}
+
+func (c *SolveProblem) Usage() string {
+	return "Suggest problem solving"
+}
+
+func (c *SolveProblem) Flags() []cli.Flag {
+	//return []cli.Flag{
+	//	commands_common.InputFileFlag(),
+	//	commands_common.ContextFlag(),
+	//	commands_common.PromptFlag(),
+	//	commands_common.OutputFlag(),
+	//	commands_common.GoogleDriveFilesFlag(),
+	//}
+	return []cli.Flag{
+
```

Comment:
why not to use flags from commands_common?

Reply:
no need file as mandatory. but it could be so it is optional. also, promt is optional. because promt could be in a file